backend/src/apiserver/server/test/arguments_zip/arguments-parameters.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Google LLC
```

Comment:
How come we need a new sample for a new format? It seems the same one could be used for all formats.

Reply:
i was having the source file and zipped file side by side so reader dont need to unzip file to know what is inside. but i think it's not necessary. removed